src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1308,12 +1479,13 @@ See the `Content Collections` guide for examples of [querying collection entries
 
 ### `getEntries()`
 
-<p><Since v="2.5.0" /></p>
+<p>
 
 **Types:**
-
 - `(Array<{ collection: string, id: string }>) => Array<CollectionEntry<collection>>`
 - `(Array<{ collection: string, slug: string }>) => Array<CollectionEntry<collection>>`
+<Since v="2.5.0" />
+</p>
```

Comment:
Like above: I'm not sure about this one. Since we have a list of types the rendering might seem odd. I tried to place `<Since />` before `Types:` but we have a new line between the two. I think the Markdown parser ignores the `<p>` tag around because of the list inside. So I'm not sure what is the best option here.

Reply:
Ah, regarding the list nesting, we might need to use HTML for the whole markup potentially? Which is a bit painful. If indeed `<ul>` is valid inside `<p>`. I think so, but let me double check.

Reply:
@delucis https://caninclude.glitch.me/caninclude?child=ul&parent=p It seems we cannot do this. 🫤

Reply:
No, `<ul>` is indeed not valid inside a `<p>`, which should only contain phrasing content. Given this, maybe it’s best to flip the order but still not squash everything together into a single paragraph so spacing is consistent.